Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
@greg0ire and @core23 what should I do?

Move ``array_flip`` to ``ColorSelectorType->configureOptions()`` ?